index.js

```diff
@@ -1 +1,72 @@
-// implement your API here
+const express = require('express');
+const cors = require('cors');
+const db = require('./data/db');
+const server = express();
+
+server.use(express.json());
+server.use(cors());
```

Comment:
Excellent, but another tip that you might like for optimization: I like to separate my imports/requires by concerns, aka express/server (`building` server) at the top, then inter-app imports separated by an empty line, then middleware imports separated by an empty line, then using middleware down another line, etc. not necessary but helps mentally map what each piece does.